CONTRIBUTING.md

```diff
@@ -36,6 +36,9 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list at](https://lists.lfedge.org/g/eve-tsc/)
+and also susbscribe to the [eve-tsc Calendar at](https://lists.lfedge.org/g/eve-tsc/calendar)
+
```

Comment:
Also specify the slack channels #eve and #eve-help at https://lists.lfedge.org/g/eve and provide a link to the wiki (https://wiki.lfedge.org/display/EVE/EVE)

FWIW I found those at the bottom of https://www.lfedge.org/projects/eve/